test/insertTest.js

```diff
@@ -1,59 +1,70 @@
-const db = require("../db")
-const assert = require('chai').assert
+var db = require('../db')
+var expect = require('chai').expect
 
-soldierObj = {
-    id: "1",
-    name: "Maya",
-    rank: "private", 
-    limitiations: ["Korona", "Weather"],
-    duties: "123456"
+var collectionName, path, soldierId
+var done = function(err, id) {id = soldierId}
+var soldierObj = {
```

Comment:
this area is for the code. all data cluters it, move into relevant test data file